shells/planner-shell/single-user-context.js

```diff
@@ -192,6 +192,7 @@ export const SingleUserContext = class {
     // TODO(sjmiles): cache and return promises in case of re-entrancy
     let promise = this.pendingStores[id];
     if (!promise) {
+      // eslint-disable-next-line no-async-promise-executor
       promise = new Promise(async (resolve) => {
         const store = await context.findStoreById(id);
```

Comment:
```
context.findStoreById(id).then(store => {
  if (store) {
    resolve(store);
  } else {
    context.createStore(type, name, id, tags).then(store => resolve(store));
  }
}
```
WDYT?

Reply:
afaict findStoreById in manifest.ts is not async, why do we need await here?

Reply:
Replaced with a simple async function (which returns a promise)